examples/getting-started-app/README.md

```diff
@@ -481,9 +486,70 @@ If this value is unset, the failover status won't be affected.
 
 ### Stateful failover
 
-The full example for [stateful failover](https://www.tarantool.io/en/doc/2.2/book/cartridge/cartridge_api/topics/failover.md#stateful-failover)
-is coming soon.
-Now you can [read the doc](../../README.md#stateful).
+**Note** that stateful failover is supported since `Cartridge` 2.1.0.
+
+[Read the doc](../../README.md#stateful) to learn more about stateful failover.
+
+First, let's set failover mode to stateful:
+
+```yaml
+---
+all:
+  vars:
+    ...
+    cartridge_failover_params:
+      mode: stateful
+      state_provider: stateboard
+      stateboard_params:
+        uri: 172.19.0.2:3310
+        password: secret-stateboard
+    ...
+```
+
+Now, check the `Failover` button in the Admin UI:
+
+![](./images/stateful-failover.png)
+
+Stateful failover was enabled, but we didn't start the Stateboard instance.
+If you click the `Issues` button, you can see that there are problems with
+state provider availability.
+
+![](./images/issues.png)
+
+Now we need to start [Stateboard instance](../../README.md#stateboard-instance).
+
+Add host marked as `stateboard` to the first machine:
+
+```yaml
+---
+all:
+  vars:
+  ...
+  hosts:
+    ...
+    my-stateboard:
+      config:
+        listen: 172.19.0.2:3310
+        password: secret-stateboard
```

Comment:
Please add some optional parameter to readme. Lock delay or custom proc title?

Reply:
I don't think that this is a good idea. I sure that this example should be as simple as possible.